nautobot/dcim/templates/dcim/location.html

```diff
@@ -128,6 +128,14 @@
                 <td>{{ object.contact_email|hyperlinked_email }}</td>
             </tr>
         </table>
+        {% with request.path|add:"?tab=contacts"|urlencode as return_url %}
+            <div class="panel-footer text-right noprint">
+                <a href="{% url 'extras:map_contact_from_location' %}?return_url={{return_url}}&location={{ object.pk }}&name={{object.contact_name}}&phone={{object.contact_phone}}&email={{object.contact_email}}&address={{object.physical_address}}&associated_object_id={{object.id}}&associated_object_type={{content_type.id}}" class="btn btn-primary btn-xs">
+                    <span class="mdi mdi-map-marker" aria-hidden="true"></span>
```

Comment:
Perhaps:

```suggestion
                    <span class="mdi mdi-account-edit" aria-hidden="true"></span>
```

as this isn't a "map" in the spatial sense.